app/decorators/proposal_decorator.rb

```diff
@@ -31,6 +31,20 @@ def approvals_in_list_order
     end
   end
 
+  def subscribers_list
+    userXrole = object.users.map {|u| [u, Role.new(u, object)] }
```

Comment:
I intentionally removed `user` from the `Role` interface, but I can see an argument for adding it back. Will do